packages/core/core/src/utils/startup-logger.ts

```diff
@@ -71,17 +78,13 @@ export default (app: Strapi) => {
     },
 
     logStartupMessage({ isInitialized }: { isInitialized: boolean }) {
-      // Should the startup message be displayed?
-      const hideStartupMessage = process.env.STRAPI_HIDE_STARTUP_MESSAGE
-        ? process.env.STRAPI_HIDE_STARTUP_MESSAGE === 'true'
-        : false;
-
-      if (hideStartupMessage === false) {
-        if (!isInitialized) {
-          this.logFirstStartupMessage();
-        } else {
-          this.logDefaultStartupMessage();
-        }
+      if (!strapi.config.get('server.logger.startup.enabled')) {
```

Comment:
Since this condition is checked in the individual methods, do we really need to keep this here?

Reply:
Eh, not necessary, but it made sense to check everywhere to prevent someone accidentally logging something they shouldn't in the future.